playbooks/responses/how-it-works.md

```diff
@@ -0,0 +1,6 @@
+Open source is a significantly volunteer-driven and collaborative system of production. Because of this structure, time and energy to get things done is the scarcest of resources in open source projects. We want Electron to be the best that it can be, but we also have to be realistic about what the team maintaining Electron can achieve. This means that:
+
+* We may not give you an ETA for when an issue will get investigated or fixed
```

Comment:
```suggestion
* We might not have an ETA for when an issue will get investigated or fixed.
```

Softens the tone a little. I'm a little torn on these bullet point changes -- this softer voice is the one I'd personally use, but I'm not sure it's congruent with the message's warning nature. So these three are more of a for-your-consideration